inc/class-wpseo-statistics.php

```diff
@@ -0,0 +1,135 @@
+<?php
+/**
+ * @package WPSEO\Internals
+ */
+
+/**
+ * Class that generates interesting statistics about things
+ */
+class WPSEO_Statistics {
+
+	/**
+	 * Returns the amount of posts that have no focus keyword
+	 *
+	 * @return int
+	 */
+	public function get_no_focus_post_count() {
+		return $this->get_post_count( 'no_focus' );
```

Comment:
This string of `no_focus` is used multiple times. Better to assign it to a class constant (`const`), and reference that where needed. As a non-string token, code editors can autocomplete, syntax highlight, and also check for typos etc.

Apply to the other strings used in the class too.